<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>751</width>
     <height>371</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox_listSoure">
      <property name="title">
       <string>listSoure</string>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>161</width>
         <height>331</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>剪切</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>复制</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>黏贴</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>粗体</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>斜体</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>下划线</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>红色</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_listWidget">
      <property name="title">
       <string>listWidget</string>
      </property>
      <widget class="QListWidget" name="listWidget_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>331</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>剪切</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_treeWidget">
      <property name="title">
       <string>treeWidget</string>
      </property>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>161</width>
         <height>331</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>编辑</string>
        </property>
       </column>
       <item>
        <property name="text">
         <string>编辑</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>格式</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>剪切</string>
        </property>
       </item>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_tableWidget">
      <property name="title">
       <string>tableWidget</string>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>161</width>
         <height>331</height>
        </rect>
       </property>
       <row/>
       <row/>
       <row/>
       <column/>
       <column/>
       <column/>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>751</width>
     <height>181</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>设置对象</string>
      </property>
      <widget class="QRadioButton" name="radioButton_listSoure">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>listSoure</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_listWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>60</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>listWidget</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_treeWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>treeWidget</string>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton_tableWidget">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>130</y>
         <width>118</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>tablewidget</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string>拖动参数设置</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QCheckBox" name="checkBox_accecptDrops">
         <property name="text">
          <string>accecptDrops</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QCheckBox" name="checkBox_dragEnable">
         <property name="text">
          <string>dragEnable</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>dragDropMode</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="comboBox_dragDropMode">
         <item>
          <property name="text">
           <string> NoDragDrop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string> DragOnly</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>DropOnly</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string> DragDrop</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string/>
          </property>
         </item>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>defultDropAction</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="comboBox_defultDropAction">
         <item>
          <property name="text">
           <string>CopyAction</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>MoveAction</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>LinkAction</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>158</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
